.github/dependabot.yml

```diff
@@ -116,14 +116,28 @@ updates:
     schedule:
       interval: daily
   - package-ecosystem: docker
-    directory: /docker-compose/elasticsearch
+    directory: /docker-compose/elasticsearch/v7
     schedule:
       interval: daily
     allow:
       - dependency-name: "docker.elastic.co/elasticsearch/elasticsearch"
         update-types: ["version-update:semver-minor"]
   - package-ecosystem: docker
-    directory: /docker-compose/opensearch
+    directory: /docker-compose/elasticsearch/v8
+    schedule:
+      interval: daily
+    allow:
+      - dependency-name: "docker.elastic.co/elasticsearch/elasticsearch"
+        update-types: ["version-update:semver-minor"]
```

Comment:
I am seeing these errors in the dependabot tab
```
The property '#/updates/20/allow/0' contains additional properties ["update-types"] outside of the schema when none are allowed
The property '#/updates/21/allow/0' contains additional properties ["update-types"] outside of the schema when none are allowed
The property '#/updates/22/allow/0' contains additional properties ["update-types"] outside of the schema when none are allowed
The property '#/updates/23/allow/0' contains additional properties ["update-types"] outside of the schema when none are allowed
```

Reply:
I see this defined differently in dependabot's own file: https://github.com/dependabot/dependabot-core/blob/1aac2b839744c809d52043fa51f447d71fe9ca48/.github/dependabot.yml#L49

Reply:
The current one is also there: https://github.com/dependabot/dependabot-core/blob/1aac2b839744c809d52043fa51f447d71fe9ca48/.github/dependabot.yml#L49